package/gluon-web-offline-ssid/Makefile

```diff
@@ -0,0 +1,13 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-web-offline-ssid
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/$(PKG_NAME)
+  TITLE:=adds an option to enable and disable the offline-ssid changer in advanced config mode
+  DEPENDS:=+gluon-core +micrond +gluon-offline-ssid
```

Comment:
https://github.com/Freifunk-Nord/gluon-ssid-changer/commit/f6db033d6568ea27805c9694b20085b8dec5ba87#diff-2e2f87a246ba417fe54fa668193fa2db

Reply:
added that fix and rebased to current master

Reply:
i think you changed it in the wrong Makefile!

Reply:
oh yes, fixed an rebased